### PR TITLE
Ask for Gradle version with bug reports

### PR DESCRIPTION
**Pull request by 3flex**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -24,5 +24,6 @@ Please search in the [issues](https://github.com/arturbosch/detekt/issues) befor
 ## Your Environment
 <!--- Include as many relevant details about the environment you experienced the bug in -->
 * Version of detekt used:
```
[the remark this answers is not in the document]
Comment (3flex) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've only ever seen it written lower case... @arturbosch could confirm for us!

[the remark this answers is not in the document]
Comment (vanniktech) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've always written it with an upper case letter

[the remark this answers is not in the document]
Comment (Mauin) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Readme.md has detekt in lower case

[the remark this answers is not in the document]
Comment (arturbosch) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I have always written it lowercase :O

**Pull request merged**